sdk/entra/Microsoft.Azure.WebJobs.Extensions.AuthenticationEvents/src/Helpers.cs

```diff
@@ -68,7 +68,12 @@ internal static HttpResponseMessage HttpErrorResponse(Exception ex)
             return response;
         }
 
-        internal static string GetFailedRequestPayload(Exception ex)
+        /// <summary>
+        /// Joins the exception messages into a json payload.
+        /// </summary>
+        /// <param name="ex">The exception thrown</param>
```

Comment:
It is not obvious that exception is allowed to be null and what method would do if it is null. Please add this information to the comment.

Reply:
Updated